README.md

```diff
@@ -35,12 +35,24 @@ args:
 dirs:
   - var/data/tmp
   - var/log
+# OPTIONAL - A map of configurations of sub-processes to launch
+sub-processes:
```

Comment:
think you've moved this to camelCase in the code itself, update here as well

Reply:
Thanks, not sure how Goland missed this.